# HG changeset patch
# User Rik <rik@octave.org>
# Date 1351617874 25200
#      Tue Oct 30 10:24:34 2012 -0700
# Node ID 38e64d9f0fda72c6ac7425be12523b129d9f54f3
# Parent  e05eafe7105aa3373938fc193d202bf8e3730987
doc: Workaround text overlap in pdf documentation caused by bug in Texinfo.

* oop.txi: Add plaintext sentence between @node and @float in section 34.4.2
to work around a bug in Texinfo causing text overlap.

diff --git a/doc/interpreter/oop.txi b/doc/interpreter/oop.txi
--- a/doc/interpreter/oop.txi
+++ b/doc/interpreter/oop.txi
@@ -490,16 +490,23 @@ in the use of methods of other classes w
 @cindex quotient
 @cindex relational operators
 @cindex subtraction
 @cindex tests for equality
 @cindex transpose
 @cindex transpose, complex-conjugate
 @cindex unary minus
 
+@c Need at least one plaintext sentence here between the @node and @float
+@c table below or the two will overlap due to a bug in Texinfo. 
+@c This is not our fault; this *is* a ridiculous kluge.
+The following table shows, for each built-in numerical operation, the
+corresponding function name to use when providing an overloaded method for a
+user class.
+
 @float Table,tab:overload_ops
 @opindex +
 @opindex -
 @opindex .*
 @opindex *
 @opindex ./
 @opindex /
 @opindex .\
